src/main/java/frc/robot/commands/LevelRobot.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.commands;
+
+import com.ctre.phoenix.sensors.WPI_Pigeon2;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ExampleSubsystem;
+import frc.robot.subsystems.Swerve;
+public class LevelRobot extends CommandBase{
+    private final Swerve drive;
+    private final PIDController pid = new PIDController(0.1, 0.001, 0.005);
+    private final PIDController headingPid = new PIDController(0.04, 0, 0.0025);
+    private double startingYaw;
+    
+    public LevelRobot(Swerve drivetrain){
+        drive = drivetrain;
+        addRequirements(drive);
+    }
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+        pid.setTolerance(0);
+        pid.setSetpoint(0);
+        startingYaw = drive.getYawAsDouble() % 360;
+        System.out.println("Command Init");
```

Comment:
Ditch the prints, or make sure they're more specific so they make sense when you see them in the log latter. This one is quite vague!

Reply:
Fixed!